src/main/java/business/UsuarioService.java

```diff
@@ -24,8 +26,8 @@
         return items;
     }
 
-    public Usuario findOne(Long id){
-        return repository.findById(id).get();
+    public Optional<Usuario> findOne(Long id){
```

Comment:
Should we return Optional object ?  Why not Usuario?  Did you tried this frontend?

Reply:
Estoy retornando el **Obtional Object** para poder realizar el `get()` dentro del controller y poder utilizar el método  `.orElseThrow()` y manejar la excepción en el caso que no exísta un ID.

Reply:
Gracias.